### ISSUE TITLE
Old FSF address

### ISSUE THREAD

**Issue by ppisar** [perl-doc-cats/perlfaq#60]
LICENSE file quotes Free Software Foundation's postal address. The address is not valid anymore. Current one can be found on <http://www.gnu.org/licenses/old-licenses/gpl-1.0.txt>. Please update the license wording to deliver up-to-date address to your users.

This is a regression in perlfaq-5.021010. perlfaq-5.021009 is good.

**Comment by ppisar** [comment]
I believe this is because you used too old Dist::Zilla::Plugin::MakeMaker:

```
--- perlfaq-5.021009/LICENSE    2015-01-25 23:36:27.000000000 +0100
+++ perlfaq-5.021010/LICENSE    2015-07-28 20:30:51.000000000 +0200
@@ -22,7 +22,7 @@
                      Version 1, February 1989
 
  Copyright (C) 1989 Free Software Foundation, Inc.
- 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ 51 Franklin St, Suite 500, Boston, MA  02110-1335  USA
 
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
diff -Naur perlfaq-5.021009/Makefile.PL perlfaq-5.021010/Makefile.PL
--- perlfaq-5.021009/Makefile.PL        2015-01-25 23:36:27.000000000 +0100
+++ perlfaq-5.021010/Makefile.PL        2015-07-28 20:30:51.000000000 +0200
@@ -1,11 +1,11 @@
 
-# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.029.
+# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.019.
```

**Comment by karenetheridge** [comment]
thanks, fixed in 9337fb3.
